RxAlamofire/RxAlamofireTests/RxAlamofireTests.swift

```diff
@@ -69,4 +71,40 @@ class RxAlamofireSpec: XCTestCase {
             XCTFail("\(error)")
         }
 	}
+
+    func testProgress() {
+        do {
+            let dataRequest = try request(HTTPMethod.get, "http://myjsondata.com").toBlocking().first()!
+            let progressObservable = dataRequest.rx.progress().observeOn(MainScheduler.instance).replayAll()
+            let _ = progressObservable.connect()
+            let delegate = dataRequest.delegate as! DataTaskDelegate
+            let progressHandler = delegate.progressHandler!
```

Comment:
Unfortunately dipping into `Alamofire` internals here, but it still seems like the cleanest synchronous way to do it.